packages/components/src/components/contract-card/contract-card-items/contract-card-sell.jsx

```diff
@@ -14,24 +14,24 @@ const ContractCardSell = ({ contract_info, getCardLabels, is_sell_requested, onC
         ev.preventDefault();
     };
 
-    return (
-        should_show_sell && (
-            <React.Fragment>
-                {is_valid_to_sell ? (
-                    <Button
-                        className={classNames('dc-btn--sell', {
-                            'dc-btn--loading': is_sell_requested,
-                        })}
-                        is_disabled={is_sell_requested}
-                        text={getCardLabels().SELL}
-                        onClick={onClick}
-                        secondary
-                    />
-                ) : (
-                    <div className='dc-contract-card__no-resale-msg'>{getCardLabels().RESALE_NOT_OFFERED}</div>
-                )}
-            </React.Fragment>
-        )
+    return should_show_sell ? (
```

Comment:
@sergei-deriv Not related to your PR, But since you have changed the condition I suggest you change it to the `Return Early` pattern.

Reply:
> @sergei-deriv Not related to your PR, But since you have changed the condition I suggest you change it to the `Return Early` pattern.

I did it because with syntax **"return should_show_sell && ( JSX )"** this component returned JSX of False. But with my refactored code **"return should_show_sell ? ( JSX ) : (JSX)"** this component always returns JSX.

Reply:
@sergei-deriv You can use the `Return Early` pattern for better readability, It's still the same.

```tsx
if (!should_show_sell) return <></>;
if (!is_valid_to_sell) return <div...
return <Button...
```

It's just a suggestion tho 😅 Feel free to ignore it 🙇🏻

Reply:
> @sergei-deriv You can use the `Return Early` pattern for better readability, It's still the same.
> 
> ```tsx
> if (!should_show_sell) return <></>;
> if (!is_valid_to_sell) return <div...
> return <Button...
> ```
> 
> It's just a suggestion tho 😅 Feel free to ignore it 🙇🏻

It's really good advice, I fully agree with you. I've updated my commit.